pages/om-abakus.tsx

```diff
@@ -0,0 +1,251 @@
+        Abakus en rekke <b>komiteer</b>, <b>undergrupper</b> og{" "}
+        <b>interessegrupper</b> som jobber for å skape et godt miljø for
+        studentene.
+      </p>
+      <p>
+        Hvis du ønsker å lese mer detaljert om linjeforeningen vår, kan du finne
+        mer informasjon på vår offisielle hjemmeside{" "}
+        <Link href="https://abakus.no/pages/info-om-abakus">abakus.no</Link>.
+        Merk at du må være logget inn får å se alt, så dette kan være en god
+        anledning til å{" "}
+        <Link href="https://abakuswebkom.notion.site/Hvordan-lage-bruker-p-Abakus-no-17be056689194337a1d44865f577d536">
+          opprette en bruker på abakus.no
+        </Link>
+        .
+      </p>
+      <br />
+
+      <AboutCard title="Hovedstyret" logo="/abakus_hs.webp">
+        <p>
+          Hovedstyret er Abakus høyeste organ etter Generalforsamlingen, har
+          ansvar for linjeforeningens daglige drift og styrer linjeforeningen i
+          henhold statuttene og strategidokumentet.
+        </p>
+        <p>
+          I Hovedstyret sitter Abakus sin leder, nestleder og økonomiansvarlig,
+          samt tre gruppeansvarlige som velges av Generalforsamlingen. Hver av
+          de gruppeansvarlige har ansvar for en gruppe med komiteer.
+          Hovedstyrets oppgaver innebærer å sette opp budsjetter, holde
+          kontakten med hovedsamarbeidspartneren til Abakus, fordele penger i
+          linjeforeningen, holde kontakt med fakultetet og instituttene og
+          representere Abakus utad ved forskjellige anledninger.
+        </p>
+        <h4 className={styles.subTitle}>Roller i hovedstyret:</h4>
+        <ul>
+          <li>Leder: Henriette Bjørheim</li>
+          <li>Nestleder: Maria Karlsen</li>
+          <li>Økonomiansvarlig: Kristoffer Wetterhus</li>
+          <li>
+            Gruppeansvarlig Inter (ansvarlig for Bedkom og Fagkom): Mona Helness
+          </li>
+          <li>
+            Gruppeansvarlig Media (ansvarlig for PR, readme og Webkom): Elena
+            Willmann
+          </li>
+          <li>
+            Gruppeansvarlig Sosial (ansvarlig for Arrkom, Koskom og LaBamba):
+            Maria Hurtado Beisvåg
+          </li>
+        </ul>
+        <p>
+          <a href="https://abakus.no/pages/styrer/12">
+            Les mer om Hovedstyret på abakus.no (krever innlogging)
+          </a>
+        </p>
+      </AboutCard>
+      <br />
+
+      <h3 className={styles.subTitle}>Komiteer</h3>
+      <p>
+        En komité er en gjeng som gjør forskjellig type arbeid for å drifte
+        linjeforeningen og arrangere faglige og sosiale arrangementer for
+        medlemmene. Hver komite tar gjerne opp 7-8 medlemmer hvert år, og
+        komitemedlemskap er en god sosial plattform, også for å bli kjent på
+        tvers av trinn. Det lar deg i tillegg gjøre meningsfullt arbeid for
+        andre, som bedrer studiehverdagen til Abakus&apos; mange medlemmer
```

Comment:
Perfectly written! The paragraph really brings forth the fun in volunteering for Abakus!